packages/pwa/app/components/product-tile/index.jsx

```diff
@@ -50,80 +50,47 @@ export const Skeleton = () => {
 }
 
 /**
- * The ProductTile is a simple visual representation of a product search hit
- * object. It will show it's default image, name and price.
+ * The ProductTile is a simple visual representation of a
+ * product object. It will show it's default image, name and price.
+ * It also supports favourite products, controlled by a heart icon.
  */
 const ProductTile = (props) => {
     const intl = useIntl()
-
-    // eslint-disable-next-line react/prop-types
-    const {
-        productSearchItem,
-        // eslint-disable-next-line react/prop-types
-        staticContext,
-        onAddToWishlistClick,
-        onRemoveWishlistClick,
-        isInWishlist,
-        isWishlistLoading,
-        ...rest
-    } = props
-    const {currency, image, price, productName} = productSearchItem
-    const styles = useMultiStyleConfig('ProductTile', {isLoading: isWishlistLoading})
+    const {product, enableFavourite = false, isFavourite, onFavouriteToggle, ...rest} = props
```

Comment:
i'd like to de-couple the concept of wishlist from the components like `ProductTile`, so i intentially renamed wishlist related props to "favourite", which i think is more generic. feedback is welcomed